stac_mjx/main.py

```
@@ -71,27 +71,27 @@ def run_stac(
         )
 
-    logging.info("Running transform()")
+    logging.info("Running ik_only()")
     with open(fit_path, "rb") as file:
         fit_data = pickle.load(file)
 
     offsets = fit_data["offsets"]
 
     logging.info(f"kp_data shape: {kp_data.shape}")
-    transform_data = stac.transform_kps(kp_data, offsets)
+    ik_only_data = stac.ik_only(kp_data, offsets)
```

Ruff

Local variable `ik_only_data` is assigned to but never used

Remove assignment to unused variable `ik_only_data`

(F841)